keychain/models/keychain_backend.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+# © 2016 Akretion Sebastien Beau
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+from odoo.tools.config import config
+
+
+class KeychainBackend(models.AbstractModel):
+    _name = 'keychain.backend'
+    _backend_name = None
+
+    name = fields.Char(required=True)
+    password = fields.Char(
+        compute="_compute_password",
+        inverse="_inverse_password",
+        required=True)
+    data = fields.Serialized(
+        compute="_compute_keychain",
+        inverse="_inverse_keychain")
+
+    def _get_technical_name(self):
```

Comment:
missing api.model

Reply:
no it's api.multi with a missing self.ensure_one()